### PR TITLE
Assignment 3 and 4

### PR DESCRIPTION
**Pull request by ershimen**

### CODE REVIEW

File: Assignment4/ershimen-180305/task06.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+g.add((ns.JaneSmith, vcard.Family, Literal("Smith", datatype=XSD.string)))
+# Visualize the results
+print(g.serialize(format="turtle"))
+
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+
+# TO DO
+print("Task 6.5")
+g.add((ns.UPM, RDF.type, ns.University))
+g.add((ns.JohnSmith, vcard.Work, ns.UPM))
```
Comment (dgarijo) [inline Assignment4/ershimen-180305/task06.py:82]: vcard.Work does not exist in vcard. Create your own namespace for youw own property if you can't find one

File: Assignment4/ershimen-180305/task07.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+"""**TASK 7.2: List all individuals of "Person" with RDFLib and SPARQL (remember the subClasses)**
+
+"""
+
+# TO DO
+print("Task 7.2")
+# RDFLib
+print("RDFLib")
+for s, _, _ in g.triples((None, RDF.type, ns.Person)):
+    print(s)
```
Comment (dgarijo) [inline Assignment4/ershimen-180305/task07.py:67]: missng the subclasses of person

File: Assignment4/ershimen-180305/task07.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+print()
+
+# SPARQL
+print("SPARQL")
+q1 = prepareQuery('''
+  PREFIX ns: <http://somewhere#>
+  SELECT ?Individual WHERE { 
+    ?Individual rdf:type ns:Person.
```
Comment (dgarijo) [inline Assignment4/ershimen-180305/task07.py:76]: missing the individuals belonging to any subclass of person

File: Assignment4/ershimen-180305/task07.py [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+print()
+
+# SPARQL
+print("SPARQL")
+q1 = prepareQuery('''
+  PREFIX ns: <http://somewhere#>
+  SELECT ?Individual ?Property ?Class WHERE { 
+    ?Individual rdf:type ns:Person.
+    ?Individual ?Property ?Class
```
Comment (dgarijo) [inline Assignment4/ershimen-180305/task07.py:106]: same as above

**Review by dgarijo — CHANGES_REQUESTED** [review]

**Comment by ocorcho** [comment]
Assignment 3 can be merged.
@dgarijo for assignment 4.

File: Assignment4/ershimen-180305/task07.py [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+print("SPARQL")
+q1 = prepareQuery('''
+  PREFIX ns: <http://somewhere#>
+  SELECT ?Individual WHERE { 
+    {
+      ?Individual rdf:type ns:Person.
+    }
+    UNION
+    {
+      ?SubClass rdfs:subClassOf ns:Person.
```
Comment (dgarijo) [inline Assignment4/ershimen-180305/task07.py:84]: Use  * to retrieve the subclass hierarchy.

File: Assignment4/ershimen-180305/task07.py [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+q1 = prepareQuery('''
+  PREFIX ns: <http://somewhere#>
+  SELECT ?Individual ?Property ?Class WHERE {
+    {
+      ?Individual rdf:type ns:Person.
+    }
+    UNION
+    {
+      ?SubClass rdfs:subClassOf ns:Person.
+      ?Individual rdf:type ?SubClass.
```
Comment (dgarijo) [inline Assignment4/ershimen-180305/task07.py:125]: Same as above

**Review by dgarijo — CHANGES_REQUESTED** [review]
Small changes requested

**Pull request merged**